applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_sink.c

```diff
@@ -390,32 +390,24 @@ static bool bis_per_subgroup_parse(const struct bt_bap_base_subgroup_bis *bis, v
 
 	LOG_DBG("Channel allocation: 0x%x for BIS index %d",
 		audio_codec_info[bis->index - 1].chan_allocation, bis->index);
+	enum bt_audio_location device_location_temp;
 
-	if (CONFIG_BT_AUDIO_CONCURRENT_RX_STREAMS_MAX == 1) {
-		/* If only one concurrent stream is supported we try to get the one matching the
-		 * channel assignment in UICR
-		 */
-		enum audio_channel audio_channel_temp;
-
-		channel_assignment_get(&audio_channel_temp);
-		if (audio_channel_temp > AUDIO_CH_NUM) {
-			LOG_ERR("Invalid channel assignment");
-			return true;
-		}
+	device_location_get(&device_location_temp);
 
-		if (audio_codec_info[bis->index - 1].chan_allocation != (audio_channel_temp + 1)) {
-			LOG_WRN("BIS index %d does not match channel assignment %d", bis->index,
-				audio_channel_temp);
-			return true;
-		}
+	if (!(audio_codec_info[bis->index - 1].chan_allocation & device_location_temp)) {
+		LOG_WRN("BIS idx %d channel alloc. 0x%x does not match this device' location 0x%x",
+			bis->index, audio_codec_info[bis->index - 1].chan_allocation,
+			device_location_temp);
+		return true;
 	}
 
 	if (POPCOUNT(bis_index_bitfield) >= CONFIG_BT_AUDIO_CONCURRENT_RX_STREAMS_MAX) {
-		LOG_DBG("Maximum number of BISes reached, ignoring BIS index %d", bis->index);
+		LOG_WRN("Maximum number of BISes reached, ignoring BIS index %d", bis->index);
```

Comment:
As mentioned in my PR, this will print a warning every time you have a mono device that tries to sync to a stereo gateway (meaning our default use case)

Reply:
We do not support the headset being a mono location device.

Reply:
A mono device would mean just left or just right